db/schema.rb

```diff
@@ -120,6 +120,7 @@
     t.string   "encrypted_otp_secret_key_salt", limit: 255
     t.string   "direct_otp"
     t.datetime "direct_otp_sent_at"
+    t.boolean  "sms_otp_delivery",                         default: true
```

Comment:
How about an integer column called `otp_delivery_preference` that can either be set to `sms` or `voice` via an `enum`? I think that better describes the intent. The user can have one of two specific delivery preferences. A boolean doesn't clearly communicate the purpose IMO. If I was new to the codebase, I would think `sms_otp_delivery` being `false` would mean SMS delivery is disabled, but it wouldn't be obvious that there is an alternative delivery mechanism that would be enabled.

Besides being more clear, the biggest advantage of using an enum is it allows us to add more options in the future if necessary, whereas a boolean doesn't. Anyone using the code in the future will thank us for making it easy to change and extend.